cdap-ui/server/config/development/cdap.json

```diff
@@ -1,6 +1,6 @@
 {
   "router.ssl.bind.port": "10443",
-  "dashboard.bind.address": "0.0.0.0",
+  "dashboard.bind.address": "127.0.0.1",
```

Comment:
@elfenheart @ajainarayanan @tonybach Changed this to localhost. Since this is used for development only should not matter. Let me know if I am missing something.

Reply:
What value gets used for the sandbox distribution that someone downloads when we release a version?

Reply:
@rohitsinha54 Yes this is used only for development. Shouldn't be a problem as long as UI is started by the backend.